File: URI/1 - INICIANTE/C/1072 - Intervalo2.c
#include <stdio.h>

int main()
{
	int x, y, dentro = 0, fora = 0, num;
	
	scanf(" %d", &x);
	
	for(y = 0; y < x; y++)
	{
		scanf(" %d", &num);
		
		if(num >= 10 && num <= 20){dentro += 1;}
		else{fora += 1;}
	}
	printf("%d in\n", dentro);
	printf("%d out\n", fora);
	return 0;
}

File: URI/1 - INICIANTE/Python/1072 - Intervalo2.py
x = int(input())
dentro = fora = 0
for y in range(0, x, 1):
    num = int(input())
    if num >= 10 and num <= 20:
        dentro += 1
    else:
        fora += 1;

print('{} in'.format(dentro))
print('{} out'.format(fora))